src/js/components/ThumbsRating/ThumbsRating.js

```diff
@@ -4,24 +4,33 @@ import { LikeFill } from 'grommet-icons/icons/LikeFill';
 import { Dislike } from 'grommet-icons/icons/Dislike';
 import { DislikeFill } from 'grommet-icons/icons/DislikeFill';
 import { RadioButtonGroup } from '../RadioButtonGroup';
+import { StyledRadioButtonBox } from '../RadioButton/StyledRadioButton';
```

Comment:
lets not import the `styledRadioButtonBox` here we don't want any other extend from `radiobutton`

Reply:
we can make a new Box here. For example lets do 

```
const StyledThumbsRatingBox = styled(Box)`
.....
`;

Reply:
Yes sure let me refactor, I was not aware of this.